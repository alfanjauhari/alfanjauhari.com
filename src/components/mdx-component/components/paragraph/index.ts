export * from './paragraph';
export * from './paragraph.styles';
