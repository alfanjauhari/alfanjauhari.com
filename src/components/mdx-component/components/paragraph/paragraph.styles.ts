import { styled } from '@/theme';

export const StyledParagraph = styled('p', {
  my: '$5',
});
