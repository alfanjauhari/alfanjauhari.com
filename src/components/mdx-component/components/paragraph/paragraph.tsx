import { HTMLAttributes } from 'react';
import { StyledParagraph } from './paragraph.styles';

export type ParagraphProps = HTMLAttributes<HTMLParagraphElement>;

export function Paragraph(props: ParagraphProps) {
  return <StyledParagraph {...props} />;
}
