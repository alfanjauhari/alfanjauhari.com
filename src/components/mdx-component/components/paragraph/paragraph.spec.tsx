import { render, screen } from '@testing-library/react';
import { Paragraph } from './paragraph';

describe('MDXComponent/Paragraph', () => {
  it('should render successfuly', () => {
    render(<Paragraph>Paragraph</Paragraph>);

    expect(screen.getByText('Paragraph')).toBeVisible();
  });

  it('should match snapshot', () => {
    render(<Paragraph>Paragraph</Paragraph>);

    expect(screen.getByText('Paragraph')).toMatchSnapshot();
  });
});
