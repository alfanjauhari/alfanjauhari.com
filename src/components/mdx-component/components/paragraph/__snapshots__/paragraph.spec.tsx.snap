// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MDXComponent/Paragraph should match snapshot 1`] = `
<p
  class="c-jcoCuS"
>
  Paragraph
</p>
`;
